server/utils/placements/index.ts

```diff
@@ -123,17 +123,21 @@ export const arrivalInformation = (placement: Cas1SpaceBooking): SummaryList =>
   }
 }
 
-export const departureInformation = (placement: Cas1SpaceBooking): SummaryList => ({
-  rows: [
-    summaryRow('Expected departure date', formatDate(placement.expectedDepartureDate)),
-    summaryRow('Actual departure date', formatDate(placement.actualDepartureDate)),
-    summaryRow('Departure time', formatTime(placement.actualDepartureDate)),
-    summaryRow('Departure reason', placement.departureReason?.name),
-    summaryRow('Breach or recall', null),
-    summaryRow('Move on', placement.departureMoveOnCategory?.name),
-    summaryRow('More information', null),
-  ].filter(Boolean),
-})
+export const departureInformation = (placement: Cas1SpaceBooking): SummaryList => {
+  const departureReason = placement.departure?.parentReason || placement.departure?.reason
+  const breachOrRecall = placement.departure?.parentReason ? placement.departure?.reason : null
```

Comment:
The logic around 'Breach or recall' relies on the actual `parentReasonId` -- you won't have that until the departures PR is merged. This one will probably have to wait!

It is a similar logic for 'Move-on category' when the selected reason is 'Planned move-on', which is also missing a test here.